packages/office-addin-debugging/src/start.ts

```diff
@@ -214,7 +233,7 @@ export async function startDebugging(manifestPath: string, appType: AppType, app
     }
 
     if (devServerCommandLine) {
-        devServerPromise = runDevServer(devServerCommandLine, devServerPort);
+        devServerPromise = runDevServer(devServerCommandLine, manifestPath, devServerPort);
```

Comment:
A new optional parameter should be added to the end, so it is not a breaking api change.